README.md

```diff
@@ -97,5 +97,7 @@ The formatters used in this repo are provided by `shell.nix`:
 This repo uses `haskell-ci`, which is provided by `shell.nix`:
 
 ```shell
-haskell-ci regenerate
+haskell-ci --hlint-version '>=3.5 && <3.6' regenerate
```

Comment:
I think it would be good if you documented the decision procedure (if haskell-ci generates a failing hlint step, ask it to use a more recent version from hackage) rather than the specific versions (which will rapidly go out of date)

Reply:
https://github.com/bellroy/aws-arn/pull/17